classes/install/PKPInstall.php

```diff
@@ -99,6 +99,12 @@ public function preInstall()
             'collation' => 'utf8_general_ci',
         ];
         FacadesConfig::set('database', $config);
+        
+        // Here need to re-register the `DatabaseServiceProvider` as when the `SessionServiceProvider`
+        // register itself in the `\PKP\core\PKPContainer::registerConfiguredProviders`, it 
+        // register an instance of `\Illuminate\Database\ConnectionInterface` which contains the 
```

Comment:
*registers

Reply:
Still needs correction.

Reply:
updated. really hope typos get fixed this time .